bitrise.yml

```diff
@@ -225,6 +233,12 @@ workflows:
             CODE_SIGNING_ALLOWED=NO -maximum-parallel-testing-workers 2 -testPlan $TEST_PLAN_NAME
         - scheme: Focus
     - deploy-to-bitrise-io@1: {}
+    - slack@3:
+        run_if: ".IsBuildFailed"
+        inputs:
+        - channel: "#focus-ios-alerts"
+        - message: "The build run the testPlan: $TEST_PLAN_NAME"
```

Comment:
I will add here the build that failed, Focus or Klar to have a more informative message